e2e/scenarios/kops-simple

```diff
@@ -99,28 +122,30 @@ cd ${REPO_ROOT}
 if [[ -z "${IMAGE_TAG:-}" ]]; then
   IMAGE_TAG=$(git rev-parse --short HEAD)-$(date +%Y%m%dT%H%M%S)
 fi
-echo "IMAGE_TAG=${IMAGE_REPO}"
+echo "IMAGE_TAG=${IMAGE_TAG}"
+
 
 # Build and push cloud-controller-manager
 cd ${REPO_ROOT}
 export KO_DOCKER_REPO="${IMAGE_REPO}"
 go run github.com/google/ko@v0.12.0 build --tags ${IMAGE_TAG} --base-import-paths --push=true ./cmd/cloud-controller-manager/
```

Comment:
can you please add the tags to the go binary, it seems that you have to set an env variable

https://github.com/ko-build/ko/blob/86b6c2854f8a7c321ce1b8ea121938ce9cc79475/docs/advanced/faq.md?plain=1#L3-L14

```

[Using -ldflags](https://blog.cloudflare.com/setting-go-variables-at-compile-time/) is a common way to embed version info in go binaries (In fact, we do this for `ko`!).
Unfortunately, because `ko` wraps `go build`, it's not possible to use this flag directly; however, you can use the `GOFLAGS` environment variable instead:

GOFLAGS="-ldflags=-X=main.version=1.2.3" ko build .
```